centreon/www/include/configuration/configObject/hostgroup/formHostGroup.php

```diff
@@ -276,6 +276,8 @@ function myReplace()
         $hgObj->setValue(
             insertHostGroupInDB(isCloudPlatform: $isCloudPlatform)
         );
+        $hgs = $acl->getHostGroupAclConf(null, 'broker');
```

Comment:
```suggestion
        $hgs = $acl->getHostGroupAclConf();
```

https://github.com/centreon/centreon/blob/develop/centreon/www/class/centreonACL.class.php#L2725C58-L2771

According to method definition it seems that the first parameter is null by default and second parameters is never used 🥹  so you could just call the function without any arguments.